src/crawlee/http_clients/httpx_client.py

```diff
@@ -46,10 +46,7 @@ def _is_proxy_error(error: httpx.TransportError) -> bool:
     if isinstance(error, httpx.ProxyError):
         return True
 
-    if any(needle in str(error) for needle in ROTATE_PROXY_ERRORS):
-        return True
-
-    return False
+    return bool(any(needle in str(error) for needle in ROTATE_PROXY_ERRORS))
```

Comment:
1. bool is redundant
2. this doesn't seem more readable IMO. Do we want that rule?

Reply:
agree, reverting it, however not disabling the [rule](https://docs.astral.sh/ruff/rules/needless-bool/) completely, as sometimes it might be useful